server/controllers/items/lib/items.coffee

```diff
@@ -146,6 +146,17 @@ module.exports = items_ =
     if image? and item.pictures.length is 0 then item.pictures = [ image ]
     return item
 
+  bundleOwnersData: (res, reqUserId, items)->
+    unless items?.length > 0
```

Comment:
L149-159: this can be removed now that both server/controllers/items/recent_public.coffee and server/controllers/items/last_public.coffee use `bundleOwnersToItems = require './lib/bundle_owners_to_items'`